go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	go.elastic.co/ecszap v1.0.2
 	go.uber.org/zap v1.27.0
 	k8s.io/api v0.29.4
-	k8s.io/apimachinery v0.29.4
+	k8s.io/apimachinery v0.30.1
```

Comment:
Can we keep it at `v0.29.4`? This may cause a dependency problem in depending projects